libs/shared/lang/en.json

```diff
@@ -1,182 +1,202 @@
 {
-  "+yDOnO": {
-    "defaultMessage": "We don't have any transactions matching your query.",
-    "description": "app.transactionList.notransactions: For when we have no transactions to display for the transaction history"
-  },
-  "024I2T": {
-    "defaultMessage": "Balance",
-    "description": "shared.widgets.balanceandprofit.balance: Title for widget Balance and profit"
-  },
-  "0zfmOG": {
-    "defaultMessage": "Cancel",
-    "description": "shared.modalOperation.cancel: Text for the cancel button for the modal tool"
-  },
-  "2Rp4o1": {
+  "'+yCmHT'": {
```

Comment:
This language file is a compiled output, so probably could be removed (perhaps when reviewing translation pipeline)